src/components/elements/Checkbox/index.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
```

Comment:
I think this file has JS(ES7) with JSX. So, it sounds weird to me to call the whole thing is JSX. So, that is a convention from other projects?